service/src/main/resources/api/openapi.yml

```diff
@@ -40,20 +40,59 @@ paths:
               schema:
                 $ref: "#/components/schemas/VersionProperties"
 
+  # Unauthenticated Auth Paths
+  /auth/exchangeAuthorizationCode:
+    parameters:
+      - $ref: "#/components/parameters/AuthCode"
+    get:
+      security: []
+      summary: |
+        Exchange an authorization code for an access token and refresh token.
+      operationId: getAuthTokens
+      tags: [Authorization]
+      responses:
+        "200":
+          $ref: "#/components/responses/TokenResultResponse"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "403":
+          $ref: "#/components/responses/PermissionDenied"
+        "500":
+          $ref: "#/components/responses/ServerError"
+
+  /auth/refreshToken:
+    parameters:
+      - $ref: "#/components/parameters/RefreshToken"
+    get:
+      security: []
```

Comment:
it's hard to see with the format. you just added this?

Reply:
and removed the "api" prefix from the paths

Reply:
ah duh.